tests/unit/test_runner.py

```diff
@@ -82,7 +82,20 @@ def test_create(
     """
 
     runner.create("test_image", vm_resources, binary_path, token)
-    assert len(pylxd.instances.all()) == 1
+
+    instances = pylxd.instances.all()
+    assert len(instances) == 1
+
+    if runner.config.proxies:
+        instance = instances[0]
+        env_proxy = instance.files.get("/opt/github-runner/.env")
+        systemd_docker_proxy = instance.files.get(
+            "/etc/systemd/system/docker.service.d/http-proxy.conf"
+        )
+        # Test the file has being written to.  This value does not contain the string as the
```

Comment:
Small typo "Test the file has been written"

Reply:
I will fix this in a upcoming PR.